README.rst

```diff
@@ -101,17 +101,37 @@ Running Tests
 Setup
 #####
 
-Tox, our test runner, tests against both Python 2.7 and Python 3.4
-environments. We have a set of tests in the :code:`dcos` package (root
-directory) and in the :code:`dcoscli` package (:code:`cli` directory). When
-running the tests describe below change directory to one of those two and
-follow the instructions.
+Tox, our test runner, tests against Python 3.4. We have a set of tests in
+the :code:`dcos` package (root directory) and in the :code:`dcoscli` package
+(:code:`cli` directory). When running the tests describe below change
+directory to one of those two and follow the instructions.
+
+
+Initialization
+#######
+
+The `config` integration tests use static config files. To run these tests
+make sure you set owner only permissions on these files:
+
+:code:`chmod 600 cli/tests/data/dcos.toml`
```

Comment:
what prevents us from checking in the file with the correct permissions.  I like that we documented this... however I would love for it to be automated.

If there is reason to not check in a file with the correct attributes, then can we have a script that automate this?